frontend/src/components/DropboxChooser/components/Content.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FC, useEffect } from "react";
+import dropins from "./dropins";
+
+// (thuang): My personal Dropbox app key. Will be replaced with
+// Single Cell app key once the contract goes through
+const TEMP_APP_KEY = "66mv8tdrtxi2s0m";
```

Comment:
Are there any security risks to having this exposed?

Reply:
Great question! I'm thinking since the app-key is not a secret (publicly available in the rendered HTML), I think it should be fine. On Dropbox's tutorial, the key is hard coded in the `<script />` tag as well :D The dangerous part of configs is done on Dropbox Developer website!